gitbook-docs/data_model_metadata.md

```diff
@@ -44,18 +44,50 @@ In the example `meta` object above, a definition is provided for an analog RPM g
 few different options for the consumer to use to display the name of the measurement and explicitly calls out the unit
 of measure. It also specifies a recommended display format via `gaugeType`.
 
+###`timeout`
+
 The `timeout` property tells the consumer how long it should consider the value valid. This value is specified
-in seconds, so for a high speed GPS sensor it may 0.1 or even 0.05. The `alertMethod`, `warnMethod`, `alarmMethod` and
-`emergencyMethod` properties tell the consumer how it should respond to an abnormal data condition. Presently the
-values for these properties are `sound` and `visual` and the method is specified as an array containing one or both of
-these options. It is up to the consumer to decide how to convey these alerts.
+in seconds, so for a high speed GPS sensor it may 0.1 or even 0.05.
+
+###`alertMethod`, etc
+The `alertMethod`, `warnMethod`, `alarmMethod` and `emergencyMethod` properties tell the consumer how it should respond to an
+abnormal data condition. Presently the values for these properties are `sound` and `visual` and the method is specified as an
+array containing one or both of these options. It is up to the consumer to decide how to convey these alerts.
 
+###`zones`
 The last property in the `meta` object is the `zones` array. This provides a series of hints to the consumer which can
-be used to properly set a range on a display gauge and also color sectors of the gauge to indicate nominal or dangerous
+be used to properly set a range on a display gauge and also color sectors of a gauge to indicate normal or dangerous
 operating conditions. It also tells the consumer which state the data is in for a given range. Combined with the alert
 method properties, all Signal K consumers can react the same way to a given state.
 
-It is also possible for a Signal K server to use this information to monitor any data which has a `meta` object and
+The possible states in ascending order of severity are:
+
```

Comment:
Nice table & order, but I am not too crazy about going so far into specifying UI specifics as to state colors that should be used. My vote is on removing the third column. There are so many ways you can make readings more or less prominent (color, size, blinking, movement, outlines,  opacity to list a few off the top of my head) that I feel we should leave all of that to the UI designer and implementor.

Capitalize the heading texts?